File: EPRO.Library.v3.5/EPRO.Library.v3.5/Modules/basMain.cs
using CODERiT.Logger.v._3._5;

namespace ELibrary.Standard.Modules
{
    internal static class basMain
    {
        private static Log1 _LogFile = new Log1(typeof(basMain)).Load(Log1.Modes.FILE, true);

        public static Log1 MyLogFile
        {
            get
            {
                return _LogFile;
            }
        }
    }
}

File: EPRO.Library.v3.5/EPRO.Library.v3.5/Modules/basMain.vb
Imports CODERiT.Logger.v._3._5


Namespace Modules


    Friend Module basMain

        Dim _LogFile As Log1 = New Log1(GetType(basMain)).Load(Log1.Modes.FILE, True)

        Public ReadOnly Property MyLogFile() As Log1
            Get
                Return _LogFile
            End Get
        End Property

    End Module

End Namespace